packages/react-app/src/styles/homepage.css

```diff
@@ -95,14 +95,14 @@
   margin-top: 1.25rem;
 }
 
-@media screen and (max-width: 720px) {
+@media screen and (max-width: 1400px) {
```

Comment:
1439

Reply:
Resolving the conversation, as we discussed in DM's that changing them causes the UI to break 🙌